encoding/wkb/sql.go

```diff
@@ -67,6 +77,32 @@ type GeometryCollection struct {
 	opts []wkbcommon.WKBOption
 }
 
+// Scan scans from a []byte.
+func (g *Geom) Scan(src interface{}) error {
+	b, ok := src.([]byte)
+	if !ok {
+		return ErrExpectedByteSlice{Value: src}
+	}
+	// NOTE(tb) other Scanners do not check the len of b, is it really useful ?
```

Comment:
I need to double check this. Specifically, this relates to how `NULL` `GEOMETRY`s and `GEOGRAPHIES` should be handled. `database/sql` has a selection of `Null*` types (e.g. `NullBool`, `NullString` etc.) and it's not yet clear to me whether we should include `NULL` handling in `wkb.Geom` by default or if we should add a separate `wkb.NullGeom`.

tl;dr please leave this in for the moment.